python/gui/qgsexpressionselectiondialog.sip

```diff
@@ -35,11 +35,25 @@ class QgsExpressionSelectionDialog : QDialog
      */
     void setGeomCalculator( const QgsDistanceArea & da );
 
+    /** Sets the message bar to display feedback from the dialog. This is used when zooming to
+     * features to display the count of selected features.
+     * @param messageBar target message bar
+     * @note added in QGIS 3.0
+     */
+    void setMessageBar( QgsMessageBar* messageBar );
+
+    /**
+     * Sets a map canvas associated with the dialog.
+     * @note added in QGIS 3.0
+     */
+    void setMapCanvas( QgsMapCanvas* canvas );
+
   public slots:
     void on_mActionSelect_triggered();
     void on_mActionAddToSelection_triggered();
     void on_mActionRemoveFromSelection_triggered();
     void on_mActionSelectIntersect_triggered();
+    void on_mButtonZoomToSelection_clicked();
```

Comment:
I think this should be private - I don't like how all these slots are public and become part of stable api for no good reason.